### PR TITLE
feat: introduce yamlfmt and apply formatting

### PR DESCRIPTION
**Pull request by c3ppc3pp**
This PR adds yaml formatter.

### CODE REVIEW

File: golang/manifest/kubernetes/configmap.yaml [hunk 1]
```diff
@@ -1,13 +0,0 @@
-# deprecated, delete these after 0.4 is out
```
Comment (c3ppc3pp) [inline golang/manifest/kubernetes/configmap.yaml:1]: We need to talk about this.

Comment (polaroi8d) [reply golang/manifest/kubernetes/configmap.yaml:1]: When?

File: Makefile [hunk 2]
```diff
@@ -143,6 +143,10 @@ test-integration:
 	cd golang && \
 	make test-integration
 
+.PHONY: yamlfmt
```
[the remark this answers is not in the document]
Comment (c3ppc3pp) [reply Makefile]: Good point! Fixed.

**Review by nandor-magyar — APPROVED** [review]
It feels clean, LGTM!

**Comment by nandor-magyar** [comment]
A workflow step could be added though to check if PR was formatted properly.

**Pull request merged**